src/app/header/components/TermButtons.tsx

```diff
@@ -0,0 +1,67 @@
+import { Button, ButtonGroup } from '@chakra-ui/react';
+import React, { useCallback, useContext, useMemo } from 'react';
+
+import { useTerm } from '../../context/TermContext';
+import { getButtonTerms, getTerms } from '../shared/utils';
+
+export function TermButtons() {
+  const [term, setTerm] = useTerm();
+  const status = useMemo(
+    () => ({
+      first: true,
+      second: false,
+      third: false,
+    }),
+    []
+  );
+
+  const termNames = getButtonTerms();
+  const terms = getTerms();
+
+  const onClick = useCallback(
+    (event: React.MouseEvent<HTMLElement, MouseEvent>, name: string) => {
+      event.preventDefault();
+      setTerm(name);
+      if (name === terms[0] && !status.first) {
+        status.first = true;
+        status.second = false;
+        status.third = false;
+      } else if (name === terms[1] && !status.second) {
+        status.first = false;
+        status.second = true;
+        status.third = false;
+      } else if (name === terms[2] && !status.third) {
+        status.first = false;
+        status.second = false;
+        status.third = true;
+      }
+    },
+    [setTerm, status, terms]
+  );
+
+  return (
+    <ButtonGroup spacing="0" isAttached>
+      <Button
+        name={terms[0]}
+        isActive={status.first}
+        onClick={(e) => onClick(e, terms[0])}
+        size="sm"
+        borderRadius="2px"
+      >
```

Comment:
I feel we should be able to iterate the buttons being rendered if we switch to an array of terms hopefully.